views/components/Graphs/BarGraphTooltip.jsx

```diff
@@ -7,11 +7,19 @@ const BarGraphTooltip = ({ classes, active, payload, label, studyTotals }) => {
     return null
   }
 
+  const isTargetShown = (chartPayload) =>
+    chartPayload.some(
+      ({ name: siteName, payload: sitePayload }) =>
+        !!sitePayload.targets[siteName]
+    )
+
   return (
     <div className={classes.tooltipContainer}>
       <div className={classes.tooltipHeaderRow}>
         <div className={classes.tooltipLabelColumn}>{label}</div>
-        <div className={classes.tooltipValueColumn}>Value / Target</div>
+        <div className={classes.tooltipValueColumn}>
+          Value {isTargetShown(payload) ? '/ Target' : null}
```

Comment:
Minor nit to make sure the spacing is accurate in both cases.

```suggestion
          {isTargetShown(payload) ? 'Value / Target' : 'Value'}
```

Reply:
Thank you, this is better than returning just null.

https://github.com/AMP-SCZ/dpdash/pull/210/commits/af36a88e4c99d6f820d3ca233a683b2115dbfc04